lib/active_shipping/carriers/usps.rb

```diff
@@ -154,6 +154,11 @@ class USPS < Carrier
       "WS" => "Western Samoa"
     }
 
+    TRACKING_ODD_COUNTRY_NAMES = {
```

Comment:
This is fine. The way we solved a similar problem in shopify is defining a `name_alternates` array in the countries YAML file (c[:name]), and then we roll that up into another key https://github.com/Shopify/shopify/blob/master/app/models/country_db.rb#L133). It's a bit more of a sustainable long term approach. Ok with this too though.